vms/platformvm/vm_regression_test.go

```diff
@@ -2129,12 +2130,18 @@ func checkValidatorBlsKeyIsSet(
 	}
 
 	val, found := vals[nodeID]
-	if !found {
+	switch {
+	case !found:
 		return database.ErrNotFound
-	}
-	if val.PublicKey != expectedBlsKey {
+	case expectedBlsKey == val.PublicKey:
```

Comment:
Can we clean this up with https://pkg.go.dev/github.com/stretchr/testify/require#Equal ?

Reply:
"Pointer variable equality is determined based on the equality of the referenced values (as opposed to the memory addresses)."

Reply:
We don't have a testing struct here.